t/resolver.t

```diff
@@ -66,3 +66,28 @@ servers: 2
 2.3.4.5:6789
 --- no_error_log
 [error]
+
+
+=== TEST 3: can have ipv6 RESOLVER
+RESOLVER env variable can be IPv6 address
+--- ONLY
```

Comment:
:eyes:

Reply:
Hah! I'm going to fix the CI to catch this.

Reply:
Fixed. And verified it was failing.